python-avd/pyavd/_eos_designs/schema/schema_fragments/platform_settings.schema.yml

```
@@ -14,7 +14,7 @@ keys:
     description:
       Custom Platform settings to override the default `platform_settings`.
       This list will be prepended to the list of `platform_settings`.
-      The first entry containing `platforms` matching the `platform` node setting will be chosen.
+      The first entry containing `platforms` matching the full regex of `platform` node setting will be chosen.
```

Hmm I think this needs to be clarified actually.

Technically this is not the `platforms` list mathcing the regex.

It is the countrary.


```suggestion
      The first entry found where any regex in the `platforms` list fully matches the `platform` node setting will be chosen.
```